.github/workflows/release-sdk.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           # Not sure why we skip PyPy and musllinux builds.
           #
           # See https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
-          CIBW_SKIP: cp36-* pp* *musllinux*
+          CIBW_SKIP: cp36-* pp*
```

Comment:
Can probably also remove "cp36-" and the comment about it. We set the Python min version in `pyproject.toml`.

Reply:
yea, that works!